mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -537,11 +701,17 @@
         <GlossaryCategory value=""/>
         <GlossaryWeakArray index="0" value="Marauder"/>
         <GlossaryWeakArray index="1" value="Zergling"/>
+        <CardLayouts index="0">
+            <LayoutButtons index="12" Face="Cancel" AbilCmd="BypassArmorCancel,255" Row="2" Column="4"/>
+        </CardLayouts>
     </CUnit>
     <CUnit id="GhostNova" parent="Ghost">
         <GlossaryCategory value=""/>
         <GlossaryWeakArray index="0" value="Marauder"/>
         <GlossaryWeakArray index="1" value="Zergling"/>
+        <CardLayouts index="0">
+            <LayoutButtons index="12" Face="Cancel" AbilCmd="BypassArmorCancel,255" Row="2" Column="4"/>
```

Comment:
I am not sure how Ghost Nova skin is created, but this will NOT add the snipe cancel to the command card, if the rare Ghost occurrence really uses this unit type!

If someone knows how the Nova skin is created, please let me know :)
Currently, I would say that there is a big chance that she does not have the cancel ability